src/pybind/mgr/dashboard_v2/frontend/src/app/core/not-found/not-found.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="row">
+  <div class="col-md-12 text-center">
+    <h1>Sorry, we could not find what you were looking for</h1>
+    <h2>Instead, enjoy some cephalopods!!!</h2>
+
+    <img class="img-responsive center-block"
```

Comment:
Could you add the class "img-rounded" than the sharp edges are removed and it looks a lot nicer just by doing this.

Reply:
done